operations/addresses.md

```diff
@@ -0,0 +1,240 @@
+            "CountrySubdivisionCode": "ES-BA",
+            "Latitude": 11,
+            "Longitude": 10
+        }
+    ],
+    "Cursor": "a5347333-74c0-4ffb-82d7-af4600df5047"
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Addresses` | array of [Address](#address-items) | required | The addresses with their items. |
+| `Cursor` | string | required | Unique identifier of the last and hence oldest address item returned. This can be used in [Limitation](../guidelines/pagination.md#limitation) in a subsequent request to fetch the next batch of older address items. |
+
+#### Address items
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Id` | string | required | Unique identifier of the [Address](configuration.md#address). |
+| `AccountId` | string | required | Unique identifier of a [Company](companies.md#company) or a [Customer](customers.md#customer) within the enterprise. |
+| `AccountType` | string | required | Discriminator specifying whether the account is a [Company](companies.md#company) or a [Customer](customers.md#customer). |
+| `Line1` | string | optional | First line of the address. |
+| `Line2` | string | optional | Second line of the address. |
+| `City` | string | optional | The city. |
+| `PostalCode` | string | optional | Postal code. |
+| `CountryCode` | string | optional | ISO 3166-1 code of the [Country](countries.md#country). |
+| `CountrySubdivisionCode` | string | optional | ISO 3166-2 code of the administrative division, e.g. `DE-BW`. |
+| `Latitude` | number | optional | The latitude. |
+| `Longitude` | number | optional | The longitude. |
+
+## Add addresses
+
+Adds a new address to the system and assigns it to a specified account. The account has to be within the same enterprise.
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/addresses/add`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0",
+    "Addresses": [
+        {
+            "AccountId": "3db2c989-7d95-42b4-a502-a9f246db1634",
+            "Line1": "Address Line 1",
+            "Line2": "Address Line 2",
+            "City": "City",
+            "PostalCode": "10 000",
+            "CountryCode": "ES",
+            "CountrySubdivisionCode": "ES-BA",
+            "Latitude": 11,
+            "Longitude": 10
+        }
+    ]
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `Addresses` | array of [Address add items](#address-add-items), max 1000 items | required | Collection of addresses to be created. |
+
+#### Address add items
```

Comment:
I copied some stuff from outlet items and kept the wording because it sounded good with using descriptive words like Add or Update. What else would you suggest? @vitezp